syllabus/curricula/plr/task_sampler.py

```diff
@@ -138,17 +200,16 @@ def _average_gae(self, **kwargs):
         returns = kwargs["returns"]
         value_preds = kwargs["value_preds"]
 
-        advantages = returns - value_preds
+        advantages = np.abs(returns - value_preds)
 
-        return advantages.mean().item()
+        return np.mean(advantages).item()
```

Comment:
Why was this code changed? Please revert it, we don't want to modify these metrics at all

Reply:
Also this is not equivalent to the previous code